src/component-library/Tabs/Tabs.style.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+
+import { theme } from '../theme';
+
+const TabListWrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  background-color: ${theme.tabs.bg};
+  padding: ${theme.spacing.spacing1};
+  // TODO: There is not a variable for this
+  border-radius: 10px;
```

Comment:
I need to clarify with @tomjeatt, how should I include this in the current theme